CMakeLists.txt

```diff
@@ -53,13 +53,16 @@ set(conky_sources ${CMAKE_BINARY_DIR}/config.h ${CMAKE_BINARY_DIR}/build.h)
 add_subdirectory(3rdparty)
 set(conky_libs ${conky_libs} toluapp_lib_static)
 set(conky_libs ${conky_libs} Vc)
+set(conky_libs ${conky_libs} PkgConfig::LIBGIT2)
+
```

Comment:
Lua and Vc are added here because they're in thirdparty directory, but all the dependencies pulled from build system are added to `conky_libs` from `cmake/ConkyPlatformChecks.cmake` file. Search for `BUILD_IMLIB2` in CMake files for an example how to add libgit dependency.